### ISSUE TITLE
Error Running npm install

### ISSUE THREAD

**Issue by ppham01** [insidesherpa/JPMC-tech-task-2-PY3#582]
Hi there, I am receiving the following error when trying to run npm install. I am on macOS Monterey 12.4, 16" M1 Pro MacBook Pro. Please advise on the steps I should take.
<img width="1255" alt="Screen Shot 2022-06-15 at 8 42 57 PM" src="https://user-images.githubusercontent.com/94084025/173973384-c02306c6-16ba-4c40-a568-e636af2cbe9e.png">

**Comment by veeru004** [comment]
Hi this is related to dependencies, kindly check my repo for updated source code (only dependencies are modified) -> https://github.com/veeru004/JP-Morgan-Perspective.git

**Comment by nmashchenko** [comment]
> Hi this is related to dependencies, kindly check my repo for updated source code (only dependencies are modified) -> https://github.com/veeru004/JP-Morgan-Perspective.git

thanks!

**Comment by LoopFruits** [comment]
thank you @veeru004, your updated source code solved all my issues.

**Comment by ayushiisharmaa** [comment]
I am having the same problem, did you figure out how to get past it?

**Comment by LoopFruits** [comment]
Yeah @ayushiisharmaa, I changed the dependencies in the package.json file. The dependencies I changed we're the first 3 I believe. You should be able to see the differences. 
![JPMC package json](https://user-images.githubusercontent.com/81322778/181832422-18ede6c5-48dc-42a1-be40-d43dda17d504.png)

**Comment by kirankuyate2157** [comment]
@veeru004  please look into my issue **_#602_** 
how to solve this issue can you guide me?

**Comment by camrynfriedman** [comment]
> 

@veeru004 I am still getting errors after using your repository.
